readme/1-INSTALL.md

```diff
@@ -41,24 +41,24 @@ Install last version of the package
 sudo apt-get install li-metrics-agent
 ```
 
-Run configuration dialog. You will be asked to give the agent a name and your server metrics token. The name is only for identification in the Load Impact application. It is recomended to choose short readable name. Name and token will be written in `config-file`. You can read about advanced configuration [here](2-CONFIGURE.md).
+Run the configuration tool. You will be asked to give the agent a name and your server metrics token. The name is used for identification in the Load Impact application so it is recomended to choose a short readable name. The name and token will be written to the `config-file`. You can read about advanced configuration of Nagios and custom metrics plugins [here](2-CONFIGURE.md).
 
 ````
 sudo li-metrics-agent-config
 ````
 
-Output like `li_metrics_agent start/running, process XXXX` means that agent installed as a service and started. It will automaticly restarted after a crash or server reboot.
+Output like `li_metrics_agent start/running, process XXXX` means that the agent has properly installed as a service and started correctly. It will be automaticly restarted after a crash or server reboot.
 
-You can also press 'Check installation' button on `https://app.loadimpact.com/server-agents/load-impact` page. If new entry comes to `Monitoring tools` list then agent is succefully installed.
+You can also press the 'Check installation' button on the `https://app.loadimpact.com/server-agents/load-impact` page. If a new entry appears in the list then the agent has been succefully installed.
 ![alt tag](check_installation.png)
 
 
-If something goes wrong it's worth to check `.log` file:
+If something goes wrong it's recommended you check the `.log` file:
 ```
 tail /var/log/li_metrics_agent.log
 ```
 
-You can manage it a usual linux service
+You can manage the agent as a regular Linux service
 ```
```

Comment:
Unfortunately, almost all changes from `Ubuntu` paragraph should be applied to `Centos` paragraph as well.